crates/bevy_winit/src/state.rs

```diff
@@ -134,23 +141,25 @@ impl<T: Event> WinitAppRunnerState<T> {
     }
 }
 
+#[cfg(feature = "custom_cursor")]
 /// Identifiers for custom cursors used in caching.
 #[derive(Debug, Clone, Hash, PartialEq, Eq)]
 pub enum CustomCursorCacheKey {
-    /// u64 is used instead of `AssetId`, because `bevy_asset` can't be imported here.
-    AssetIndex(u64),
-    /// u128 is used instead of `AssetId`, because `bevy_asset` can't be imported here.
```

Comment:
why couldnt bevy_asset be imported here before?

Reply:
The original custom cursor PR calls out a circular dependency if bevy_render depends on bevy_asset. Not really sure why/where, though. But now it works :D